instrumentation/spring-cache-3.1.0/src/main/java/org/springframework/cache/Cache_Instrumentation.java

```diff
@@ -0,0 +1,42 @@
+package org.springframework.cache;
+
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.NewField;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import org.springframework.cache.Cache;
+
+import java.util.logging.Level;
+
+@Weave(type = MatchType.Interface, originalName = "org.springframework.cache.Cache")
+public abstract class Cache_Instrumentation {
+    public Cache.ValueWrapper get(Object key) {
+        Cache.ValueWrapper result = Weaver.callOriginal();
+        NewRelic.incrementCounter("Spring/Cache/" + getProviderClassName() + "/" + getName() + (result == null ? "/Misses" : "/Hits"));
+        return result;
+    }
+
+    public void evict(Object key) {
+        Weaver.callOriginal();
+        NewRelic.incrementCounter("Spring/Cache/" + getProviderClassName() + "/" + getName() + "/Evict");
```

Comment:
The operation name usually starts with a lowercase character, probably because it is the method name, which starts with lowercase.

Reply:
toLower'ed the operation names